packages/common/container-definitions/src/loader.ts

```diff
@@ -240,10 +352,10 @@ export interface IContainer extends IEventProvider<IContainerEvents>, IFluidRout
     serialize(): string;
 
     /**
-     * Get an absolute url for a provided container-relative request url.
+     * Get an absolute URL for a provided container-relative request URL.
      * If the container is not attached, this will return undefined.
      *
-     * @param relativeUrl - A container-relative request URL
+     * @param relativeUrl - A container-relative request URL.
      */
     getAbsoluteUrl(relativeUrl: string): Promise<string | undefined>;
```

Comment:
I personally recommend against using this API (common scenarios have better options), though I've not tried to exhaustively examine all known uses to confirm they all have good alternatives.  Maybe worth seeing if others think we can discourage this yet or not.